src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -634,6 +634,16 @@ $(function () {
 
     // Lightbox
     lightbox.init();
+
+    // free-range price input auto-check checkbox
+    $(".input-item-price").each(function() {
+        var input = document.getElementById("item_" + this.id.substr("price-item-".length));
```

Comment:
Should we support variations here as well to be consistent?

Reply:
Argh, that is what I forgot about to test – free price on variations. Of course we should be consistent. I will update accordingly.

Reply:
I changed it to a data-attribut that links to the ID of the element and set it onto the checkbox-element instead of the free-price input. Somehow seemed a more logical place.